python/jiminy_py/src/jiminy_py/__init__.py

```diff
@@ -15,3 +15,5 @@
 
 # Import core submodule
 from . import core
+
+from .utils import read_log
```

Comment:
Why not implement `read_log` in `log.py` ?

Reply:
By the way, `utils.py` is missing.

Reply:
I just didn't see log.py :)